src/routes/Player/styles.less

```diff
@@ -36,13 +36,14 @@ html:not(.active-slider-within) {
         left: 0;
         right: 0;
         bottom: 0;
-        z-index: 0;
 
         &.error-layer {
+            z-index: 1;
```

Comment:
whats the reason of those changes in the zindex?

Reply:
I believe it's because the video-layer happens to be on top of the error-layer, preventing the button to be clicked

Reply:
i see the issue now.
the correct way to solve this is by fixing the markup hierarchy.
revert the z-index changed.
move the `{error !== null ? </> : null}` block right after the layer with onVideoClick/onVideoDoubleClick and just before the layer that is shown when `subtitlesMenuOpen || infoMenuOpen`